### PR TITLE
Footer Navigation

### PR DESCRIPTION
**Pull request by Shubham-Sharma-77**
Footer Navigation

### CODE REVIEW

File: stories/Organism/Footer/footer.scss [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        padding-bottom: $spacing-48;
+      }
+    }
+  }
+  &__bottom {
+    &--two {
+      flex-direction: column-reverse;
+      @include devicebreak(medium) {
+        flex-direction: unset;
+      }
```
Comment (surbhi90) [inline stories/Organism/Footer/footer.scss:50]: @Shubham-Sharma-77 If you want to set the flex direction till small then you can use that media query

File: stories/Organism/Footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      p {
+        margin: 0 $spacing-16 0;
+      }
+    }
+  }
+  &__foot {
+    margin: 0 $spacing-16;
+    @include devicebreak(medium) {
+      margin: $spacing-80 0 0 0;
```
Comment (surbhi90) [inline stories/Organism/Footer/footer.scss:59]: @Shubham-Sharma-77 We can mention it like $spacing-80 0 0

File: stories/Organism/Footer/footer.scss [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    &_text {
+      margin: 0 $spacing-16 $spacing-32;
+      @include devicebreak(medium) {
+        margin: 0 0 $spacing-32 0;
+      }
+    }
+  }
+  &__icons {
+    justify-content: start;
+    margin: 0 $spacing-16 $spacing-32 $spacing-16;
```
Comment (surbhi90) [inline stories/Organism/Footer/footer.scss:76]: @Shubham-Sharma-77 we can mention it like 0 $spacing-16 $spacing-32

File: stories/assets/js/footer_custom.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      $('.footer__items li > h6').click(function () {
+        $('li').removeClass('active').find('.footer__panel').stop()
+          .slideUp();
+        $(this).closest('li').toggleClass('active').find('.footer__panel')
+          .stop()
+          .slideToggle();
+      });
+    }
+  });
+}
```
Comment (surbhi90) [inline stories/assets/js/footer_custom.js:14]: @Shubham-Sharma-77 Code is quite similar to the faq code so can we make these codes generic for accordion and use only single code at both places

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/assets/js/footer_custom.js (hunk 4 above)
Comment (Shubham-Sharma-77) [reply stories/assets/js/footer_custom.js:14]: @surbhi90 ok but how we can run the similar js on mobile screen on footer navigation.

**Pull request merged**